libs/liblobby/lobby/interface_skirmish.lua

```diff
@@ -7,25 +7,33 @@ function InterfaceSkirmish:init()
 	self.useTeamColor = true
 end
 
-function InterfaceSkirmish:WriteTable(key, value)
-	local str = '\t['..key..']\n\t{\n'
+function InterfaceSkirmish:WriteTable(key, value, tabs)
```

Comment:
What are these changes for? Script writing seems fine to me.

Reply:
It it to make it human readable for any inner elements, like aioptions